Unicode/makeutype.c

```diff
@@ -177,15 +187,53 @@ static void FigureAlternates(long index, char *apt, int normative) {
 	alts[alts[index][0]][0] = index;
 }
 
-static void processAssignment(long index,char *pt) {
+static int processAssignment(long index,char *pt,long *flg) {
     static long first=-1;
     long i;
 
-    if ( index>0x11ffff )
-return;
+    if ( index<0 ) return( -1 );
+    if ( index>0x11ffff ) return( 0 );
     ++pt;							/* move past semicolon */
```

Comment:
That's not what I meant. What I meant was that there is a potential segfault here if argument pt is a zero-length string because it gets incremented past the first value before it gets used. All we need is `if (pt[0] == '\0') return -1`, and I'm hoping that you can just put it into this patch.